src/main/java/com/ably/kafka/connect/mapping/ChannelSinkMapping.java

```diff
@@ -16,5 +16,8 @@ public interface ChannelSinkMapping {
      * @param ablyRealtime AblyRealtime instance.
      * @return The channel.
      */
-    Channel getChannel(@Nonnull SinkRecord sinkRecord,@Nonnull AblyRealtime ablyRealtime) throws AblyException, ChannelSinkConnectorConfig.ConfigException;
+    Channel getChannel(@Nonnull SinkRecord sinkRecord,@Nonnull AblyRealtime ablyRealtime) throws AblyException,
+            ChannelSinkConnectorConfig.ConfigException;
+
+    String getChannelName(@Nonnull SinkRecord sinkRecord);
```

Comment:
This interface still references an `AblyRealtime` instance, which we don't use anymore. Do we ever call `getChannel()` now? I can only see references to `getChannelName()`

We should probably just remove `getChannel()` from this interface if it's no-longer being used. It was actually added as an extensibility point for users to set up things like encryption per-channel, but I assume that's not an option using the Batch API for submission anyway, @paddybyers ?